Lombiq.HelpfulLibraries.OrchardCore/Readme.md

```diff
@@ -0,0 +1,28 @@
+# Lombiq Helpful Libraries - Orchard Core Libraries
+
+
+
+## About
+
+Various useful libraries that can be handy when developing for [Orchard Core CMS](https://orchardcore.net/).

```

Comment:
```suggestion
Various useful libraries that can be handy when developing for [Orchard Core](https://orchardcore.net/) CMS.
```

I'm not certain adding "CMS" is necessary at all. Yes, it is a CMS, but you don't write "ASP.NET Core Web Framework" either. At least it should be excluded from the link like suggested above because it's not part of the product's name. Also it could cause brand confusion since O1 is often referred to as "Orchard CMS". Same suggestion applies to other mentions of "Orchard Core CMS", like in the csproj.

Reply:
I've removed it everywhere.